src/erlcloud_aws.erl

```diff
@@ -146,7 +146,7 @@ aws_region_from_host(Host) ->
         [_, Value, _, _ | _Rest] ->
             Value;
         _ ->
-            "us-east-1"
+            application:get_env(erlcloud, aws_region, "us-east-1")
```

Comment:
do you mean `aws_config` record? I can't do that, it will break everyone, who's been relying on `aws_region_from_host`.
or do you mean I need to create a constant in the module?